go/vt/binlog/binlogplayer/binlog_player.go

```diff
@@ -263,8 +263,8 @@ func (blp *BinlogPlayer) exec(sql string) (*sqltypes.Result, error) {
 	queryStartTime := time.Now()
 	qr, err := blp.dbClient.ExecuteFetch(sql, 0, false)
 	blp.blplStats.Timings.Record(BlplQuery, queryStartTime)
-	if time.Now().Sub(queryStartTime) > SlowQueryThreshold {
-		log.Infof("SLOW QUERY '%s'", sql)
+	if d := time.Now().Sub(queryStartTime); d > SlowQueryThreshold {
+		log.Infof("SLOW QUERY (took %.2fs) '%s'", d.Seconds(), sql)
```

Comment:
I did that on purpose :)

Examples for the different outputs:

%.2f seconds:

```
  0.12s
  1.23s
 12.34s
123.45s
```

%v milliseconds:

```
   120ms
  1230ms
 12340ms
123450ms
```

I found the seconds format more readable because when the query is actually slow, it's usually slow on the order of seconds e.g. I saw 5 seconds.

Let me know if you don't agree and still want me to change it :)